SimpleRestaurantApi.md

```diff
@@ -3,13 +3,13 @@
 (See "Assignment" section for assignment details)
 
 The programming language required to complete the test will be communicated to you prior to being given the assignment.
-It may be permissable to choose between a set of languages, or you may be directed to complete the assignment using
+It may be permissible to choose between a set of languages, or you may be directed to complete the assignment using
 a prescribed programming language.
 
-Please try to spend around 4-6 hours on this test.  The focus will be on data structure choice, API design and 
```

Comment:
if you have the spare time, it might be a good idea to actually build this yourself, that way it can also give you some ideas of what you would expect the submission to look like